### ISSUE TITLE
Cannot display file list of built package with the key suggested by the French locale

### ISSUE THREAD

**Issue by calimeroteknik** [archlinuxfr/yaourt#251]
When I press `v` at the end of the build process, the files in the package are displayed, regardless of the locale.

When running with a French locale, `A` is suggested instead, but if I press it, yaourt installs the package instead of listing its files.

**Comment by larchunix** [comment]
Error with `V` should be fixed by 153c1f3. However I can't reproduce the problem with `A`:

```
$ yaourt -S bvi
...
==> Création finie : bvi 1.4.0-3 (dim. mars 13 22:36:59 CET 2016)
==> Nettoyage...

==> Continuer d'installer bvi ? [O/n]
==> [A]fficher le contenu du paquet [V]érifier avec namcap
==> ------------------------------------------------------
==> a

bvi /usr/
bvi /usr/bin/
bvi /usr/bin/bmore
bvi /usr/bin/bvedit
bvi /usr/bin/bvi
bvi /usr/bin/bview
bvi /usr/share/
bvi /usr/share/bvi/
bvi /usr/share/bvi/bmore.help
bvi /usr/share/man/
bvi /usr/share/man/man1/
bvi /usr/share/man/man1/bmore.1.gz
bvi /usr/share/man/man1/bvi.1.gz
```

**Comment by calimeroteknik** [comment]
I checked and the issue is only present in the current stable release of yaourt, yaourt-git doesn't have it.
